index.src.html

```diff
@@ -849,6 +864,54 @@ <h3 id="workers">Workers</h3>
 
 </section>
 
+<section>
+  <h2 id="premission-prompt">Permission Prompt</h2>
+
+  <em>This section is not normative.</em>
+
+  According to the discutions
+  [[Issue#23](https://github.com/WICG/private-network-access/issues/23)], the
+  <dfn export local-lt="permission prompt">private network access permission
+  prompt</dfn> is introduced to relax mixed-content checks.
+
+  Private network access permission prompt is a <a spec="permissions">permission</a>.
+
+  <h3 id="permission-goals">Goals</h3>
+
+  The overarching goal is to provide a way for private devices which are not able
+  and not welling to directly talk to any public servers including TLS certificate
+  authorities and not able to apply secure contexts to relax mixed-content checks.
```

Comment:
```suggestion
  The overarching goal is to provide a way for private devices which are not able
  or not willing to directly talk to any public servers including TLS certificate
  authorities or the ones which are not able to apply secure contexts to relax mixed-content checks.
```

Reply:
This is really unclear to me. How about:

```suggestion
  The goal of the permission is to allow communication from public websites to local
  network servers over HTTP, which would otherwise be prevented by the
  <a href="#secure-context-restriction">secure context restriction</a> and mixed
  content checks. Migrating local network servers to HTTPS has indeed proven to
  be often difficult, even sometimes impossible.
```